### ISSUE TITLE
Is there an option to use goldberg experimental? specifically to retain the original DLL of the game while still cracking it. As some games detect when their DLL is modified.

### ISSUE THREAD

**Issue by ExLuck6** [SteamAutoCracks/Steam-auto-crack#22]
If the option does exist, could you please tell me how to enable it? I can't seem to find it.

If it doesn't exist, could you please add it as an option/feature that can be toggled in "Goldberg Steam Emulator Configuration" ?

My "issue" / concern is really just

"Some games detect when their EXE / DLL has been modified, but I believe through goldberg experimental, you can keep the original DLLs while using goldberg experimental DLLS (you rename them to something like crackedsteamapi64.dll) to trick the game"

Basically, the original DLLs remain unmodified, while the "crackedsteamapi.dll" re-routes the logic of the api or something...

**Comment by ExLuck6** [comment]
The goldberg emulator master branch release got a folder called "goldberg experimental", that's the folder I am talking about.

**Comment by oureveryday** [comment]
Sorry this function does not exist, you have to manually crack it.

**Comment by oureveryday** [comment]
Now experimental Build is available in Update/Download Menu, however you must maunally apply steamclient(64).dll and coldclientloader.

**Comment by ExLuck6** [comment]
> 

Awesome! Thank you so much
